src/java.base/share/classes/jdk/internal/foreign/abi/NativeEntryPoint.java

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
This file was moved here from `jdk.internal.invoke`, but because a lot of code was remove it might be showing up as a new file.